pkg/api/cache_setting/cache_setting.go

```diff
@@ -0,0 +1,101 @@
+package cachesetting
+
+import (
+	"context"
+
+	"github.com/aziontech/azion-cli/pkg/contracts"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/aziontech/azion-cli/utils"
+	sdk "github.com/aziontech/azionapi-go-sdk/edgeapplications"
+	"go.uber.org/zap"
+)
+
+func (c *Client) Create(ctx context.Context, req *CreateRequest, applicationId int64) (Response, error) {
+	logger.Debug("Create Cache Settings")
+
+	request := c.apiClient.EdgeApplicationsCacheSettingsAPI.
+		EdgeApplicationsEdgeApplicationIdCacheSettingsPost(ctx, applicationId).
+		ApplicationCacheCreateRequest(req.ApplicationCacheCreateRequest)
+	cacheResponse, httpResp, err := request.Execute()
+	if err != nil {
+		logger.Debug("Error while creating a cache setting", zap.Error(err))
```

Comment:
We should use that utils `LogAndRewindBody` for these calls
I'll do it in my PR as well :)